commit bfa990472dae3a497c779a931ead282c431424a0 (refs/changes/30/2130/5)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-06-29T22:38:48+00:00 (2 years, 3 months ago)
    
    TOOLS-1800 Warden should have restify as a peer and dev dependency
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 3c44616..caa91da 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "restify-warden",
-  "version": "0.1.2",
+  "version": "0.2.0",
   "description": "Yet another parameter validator",
   "author": "Joyent, Inc",
   "main": "./lib/validate.js",
@@ -20,6 +20,7 @@
     "eslint": "2.13.1",
     "eslint-plugin-joyent": "1.0.1",
     "tape": "^4.2.2",
+    "restify": "^4.0.0",
     "faucet": "0.0.1"
   },
   "scripts": {
@@ -28,7 +29,9 @@
     "assert-plus": "0.1",
     "vasync": "1.6.4",
     "verror": "1.3",
-    "ip6addr": "0.2.2",
-    "restify": "git://github.com/mcavage/node-restify.git#0d7b4ba"
+    "ip6addr": "0.2.2"
+  },
+  "peerDependencies": {
+    "restify": "^4.0.0"
   }
 }
